tracer/src/Datadog.Trace/Util/RuntimeId.cs

```diff
@@ -0,0 +1,82 @@
+// <copyright file="RuntimeId.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Threading;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.Util
+{
+    internal static class RuntimeId
+    {
+#if NETFRAMEWORK
+        private const string NativeLoaderLibNameX86 = "Datadog.AutoInstrumentation.NativeLoader.x86.dll";
+        private const string NativeLoaderLibNameX64 = "Datadog.AutoInstrumentation.NativeLoader.x64.dll";
+#else
+        private const string NativeLoaderLibNameX86 = "Datadog.AutoInstrumentation.NativeLoader.x86";
+        private const string NativeLoaderLibNameX64 = "Datadog.AutoInstrumentation.NativeLoader.x64";
+#endif
+
+        private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(RuntimeId));
+        private static string _runtimeId;
+
+        public static string Get() => LazyInitializer.EnsureInitialized(ref _runtimeId, () => GetImpl());
+
+        private static string GetImpl()
+        {
+            if (TryGetRuntimeIdFromNative(out var runtimeId))
+            {
+                Log.Information("Runtime id retrieved from native: " + runtimeId);
+                return runtimeId;
+            }
+
+            var guid = Guid.NewGuid().ToString();
+            Log.Information("Unable to get the runtime id from native. Fallback to Guid.NewGuid() : {NewGuid}", guid);
```

Comment:
I think those logs should be debug instead of info, they don't bring much except for very specific cases

Reply:
will do

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/2474/commits/f4247dca6dae6fd0904e35f31f7a86a0288d319a